### PR TITLE
Update run documentation

### PR DESCRIPTION
**Pull request by cchantep**

### CODE REVIEW

File: docs/running.md [hunk 1]
```diff
@@ -32,11 +34,21 @@ docker run -v $STEWARD_DIR:/opt/scala-steward -it scala-steward:0.1.0-SNAPSHOT \
...
+
+It can also be useful to pass additional environment variables from the command line using the `--env-var` flag as shown in the examples above. You can use this to pass any credentials required by your projects to resolve any private dependencies, e.g.:
 
 ```bash
 --env-var BINTRAY_USER=username \
 --env-var BINTRAY_PASS=password
 ```
 
-These variables will be accessible (in sbt) to all of the projects that Scala Steward checks dependencies for.
+These variables will be accessible (in SBT) to all of the projects that Scala Steward checks dependencies for.
```
Comment (fthomas) [inline docs/running.md]: [It should just be sbt.](https://www.scala-sbt.org/1.x/docs/Faq.html#What+does+the+name+%E2%80%9Csbt%E2%80%9D+stand+for%2C+and+why+shouldn%E2%80%99t+it+be+written+%E2%80%9CSBT%E2%80%9D%3F) :-)
```suggestion
These variables will be accessible (in sbt) to all of the projects that Scala Steward checks dependencies for.
```

Comment (cchantep) [reply docs/running.md]: Weird naming ... updated

**Comment by fthomas** [comment]
Many thanks @cchantep! I think we can close #48 with this update.

**Comment by cchantep** [comment]
Ok, thx. Just had the [root question](https://github.com/fthomas/scala-steward/issues/48#issuecomment-510028558).

**Pull request merged**